config/settings/base.py

```diff
@@ -277,7 +277,7 @@
     "default": {
         "BACKEND": "channels_redis.core.RedisChannelLayer",
         "CONFIG": {
-            "hosts": [("127.0.0.1", 6379)],
+            "hosts": [(env("REDIS_HOST", default="127.0.0.1"), 6379)],
```

Comment:
Just making a note of this here: this and the `CELERY_BROKER_URL` should be added to the readme. I can add them after merging this in.